include/magma/app/Camera.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include "magma/glm_inc.h"
+
+class Keyboard;
+class Mouse;
+
+class Camera {
+public:
+    Camera(float zNear, float zFar, int width, int height, float FOV);
+
+    void updateScreenSize(int width, int height);
+    void update(Keyboard& keyboard, Mouse& mouse, float elapsedTime);
```

Comment:
For now this is ok, but later we need to separate input processing from camera

Reply:
Definitely yes.
Also if we'll support Android platform we'll have to make a separate logic for updates.